src/controllers/commands/CommandController.cpp

```diff
@@ -697,6 +697,33 @@ void CommandController::initialize(Settings &, Paths &paths)
             return "";
         });
 
+    this->registerCommand(
+        "/popup", [](const QStringList &words, ChannelPtr channel) {
+            QString target(words.size() < 2 ? channel->getName() : words[1]);
+
+            if (words.size() < 2 &&
+                (!channel->isTwitchChannel() || channel->isEmpty()))
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Usage: /popup <channel>. You can also use the command "
```

Comment:
```suggestion
                    "Usage: /popup [channel]. You can also use the command "
```
`[]` optional vs `<>` required - https://github.com/Chatterino/chatterino2/pull/3530#issuecomment-1019238728 for extra reference

Reply:
Actually, with the extra explanation, this one might be an actual technicality.